official/recommendation/ncf_keras_main.py

```diff
@@ -121,25 +150,31 @@ def _get_keras_model(params):
   batch_size = params['batch_size']
 
   user_input = tf.keras.layers.Input(
-      shape=(),
-      batch_size=batch_size,
+      shape=(batch_size,),
```

Comment:
I feel like this definitely needs a comment explaining.

Reply:
Done.